x-pack/platform/plugins/shared/stack_connectors/public/connector_types/cases_webhook/webhook_connectors.test.tsx

```diff
@@ -295,6 +295,81 @@ describe('CasesWebhookActionConnectorFields renders', () => {
       expect(await screen.findByTestId('horizontalStep2-current')).toBeInTheDocument();
     });
 
+    it('includes dynamic __internal__.headers fields in step1 validation', async () => {
```

Comment:
I think this title doesn't match what the test is doing. The form is valid, but we can't really tell whether validation was called.

Maybe something like:
```suggestion
    it('form submit works with valid headers', async () => {
```

And could you please also had a secret header to the array?